core/sphinx/sphinx_ctidh_test.go

```diff
@@ -31,26 +36,43 @@ func TestHybridCtidhForwardSphinx(t *testing.T) {
 	t.Parallel()
 	const testPayload = "It is the stillest words that bring on the storm.  Thoughts that come on doves’ feet guide the world."
 
-	mynike := hybrid.CTIDHX25519
+	mynike := hybrid.CTIDH1024X25519
 	g := geo.GeometryFromUserForwardPayloadLength(mynike, len(testPayload), false, 5)
 
 	t.Logf("NIKE: %s", g.NIKEName)
 	t.Logf("KEM: %s", g.KEMName)
 
-	sphinx := NewSphinx(g)
+	sphinx := NewNIKESphinx(mynike, g)
 
 	testForwardSphinx(t, mynike, sphinx, []byte(testPayload))
 }
 
+func TestSphinxConstruction(t *testing.T) {
+	var mynike nike.Scheme
+	mynike = ecdhnike.NewEcdhNike(rand.Reader)
+	g := geo.GeometryFromUserForwardPayloadLength(mynike, 12345, false, 5)
+	t.Logf("NIKEName %s", g.NIKEName)
+	sphinx := NewSphinx(g)
+	require.NotNil(t, sphinx.nike)
+
```

Comment:
Could return early here on those platforms?

Reply:
that test will not run by default unless you specify the ctidh tag... however just that tag alone is not sufficient.. there's a bunch of other env vars you need to specify in order to make it compile with CTIDH. it's a pain and I'm planning on putting all that stuff in a makefile at the root of the monorepo soon.

//go:build ctidh

Reply:
Cool, fair enough